ee/localserver/server.go

```diff
@@ -99,13 +101,16 @@ func New(logger log.Logger, db *bbolt.DB, kolideServer string) (*localServer, er
 	// for example:
 	// curl localhost:40978/query --data '{"query":"select * from kolide_launcher_info"}'
 	// mux.Handle("/query", ls.requestQueryHandler())
+	// curl localhost:40978/scheduledquery --data '{"name":"pack:kolide_device_updaters:agentprocesses-all:snapshot"}'
+	// mux.Handle("/scheduledquery", ls.requestScheduledQueryHandler())
 
 	srv := &http.Server{
 		Handler:           ls.requestLoggingHandler(ls.preflightCorsHandler(ls.rateLimitHandler(mux))),
 		ReadTimeout:       500 * time.Millisecond,
 		ReadHeaderTimeout: 50 * time.Millisecond,
-		WriteTimeout:      5 * time.Second,
-		MaxHeaderBytes:    1024,
+		// WriteTimeout very high due to retry logic in the scheduledquery endpoint
+		WriteTimeout:   30 * time.Second,
```

Comment:
too much?

Reply:
Probably fine. We can futz around as needed.